www/src/components/shell/onboarding/applications/ApplicationsSelection.tsx

```diff
@@ -0,0 +1,156 @@
+import {
+  Button,
+  Div,
+  Flex,
+  H2,
+  P,
+} from 'honorable'
+import {
+  Input,
+  LoopingLogo,
+  MagnifyingGlassIcon,
+  RepositoryChip,
+} from 'pluralsh-design-system'
+import capitalize from 'lodash/capitalize'
+import Fuse from 'fuse.js'
+
+import { APPLICATIONS_QUERY } from '../../queries'
+import { MAX_SELECTED_APPLICATIONS } from '../../constants'
+
+import OnboardingCard from '../OnboardingCard'
+
+const searchOptions = {
+  keys: ['name'],
+  threshold: 0.25,
+}
+
+type ApplicationsSelectionProps = {
+  onNext: () => void
+}
+
+function ApplicationsSelection({ onNext }: ApplicationsSelectionProps) {
+  const [search, setSearch] = useState('')
+  const [selectedApplicationIds, setSelectedApplicationIds] = useState<string[]>([])
+  const { data, loading, error } = useQuery(APPLICATIONS_QUERY)
+
+  function toggleApplication(id: string) {
+    setSelectedApplicationIds(ids => (ids.includes(id) ? ids.filter(_id => _id !== id) : [...ids, id].filter((_x, i) => i < MAX_SELECTED_APPLICATIONS)))
+  }
+
+  function handleSkipDemo() {
+
+  }
+
+  if (loading) {
+    return (
+      <Flex
+        flexGrow={1}
+        align="center"
+        justify="center"
+        padding="xlarge"
+      >
+        <LoopingLogo />
+      </Flex>
+    )
+  }
+
+  if (!data || error) {
+    return (
+      <Flex
+        flexGrow={1}
+        align="center"
+        justify="center"
+        padding="xlarge"
+      >
+        An error occured, please reload the page.
+      </Flex>
+    )
+  }
+
+  const applications = data.repositories.edges
+    .map(x => x.node)
+    .filter(x => !x.private)
+
+  const fuse = new Fuse(applications, searchOptions)
+  const filteredApplications = search ? fuse.search(search).map(x => x.item) : applications
+
+  return (
+    <OnboardingCard
+      flexGrow={1}
+      overflow="hidden"
+    >
+      <H2
+        subtitle1
+        marginTop="medium"
+      >
+        Choose applications
+      </H2>
+      <P
+        body2
+        color="text-light"
+        marginTop="xsmall"
+      >
+        Plural has over {Math.floor(applications.length / 10) * 10} open-source apps to choose from. Select up to five apps to begin.
+      </P>
+      <Input
+        autoFocus
+        value={search}
+        onChange={event => setSearch(event.target.value)}
+        placeholder="Search an application"
+        startIcon={(
+          <MagnifyingGlassIcon />
+        )}
+        width="100%"
+        marginTop="large"
+      />
+      <Div
+        marginTop="medium"
+        flexGrow={1}
+        overflowY="auto"
+        display="grid"
+        gridTemplateColumns="repeat(3, 1fr)"
+        gridTemplateRows="repeat(auto-fill, 42px)"
+        gridColumnGap="16px"
+        gridRowGap="16px"
+        paddingRight="xsmall"
+        paddingBottom="medium"
+      >
+        {filteredApplications.map(application => (
+          <RepositoryChip
+            imageUrl={application.icon}
+            label={capitalize(application.name)}
```

Comment:
It will look a bit strange for some names like `csgo` etc. I'd vote for consistency here and try to make it look the same in every view.

Reply:
@Agreed, but otherwise, LGTM.

Reply:
The design ask for it.